api/auth/auth.go

```diff
@@ -33,8 +33,8 @@ const (
 	codeChallengeLength = 87
 	sessionFileName     = "session.json"
 
-	prodSupabaseURL       = "https://fcqqkxwlntnrtjfbcioz.supabase.co"
-	prodSupabasePublicKey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImZjcXFreHdsbnRucnRqZmJjaW96Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTg0MTkyNzQsImV4cCI6MjAxMzk5NTI3NH0.ymWWYdnJC2gsnrJx4lZX2cfSOp-1xVuWFGt1Wr6zwtg"
+	prodSupabaseURL       = "https://ibcwmlhcimymasokhgvn.supabase.co"
+	prodSupabasePublicKey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImliY3dtbGhjaW15bWFzb2toZ3ZuIiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTkyOTU1MzMsImV4cCI6MjAxNDg3MTUzM30.Mr-ucuNDBjy9BC7NJzOBBi0Qz8WYiKI4n0JtWr4_woY"
```

Comment:
Note: we had the wrong public credentials here for `prod`: these are the beta environment credentials